.github/workflows/lint.yml

```diff
@@ -1,17 +1,11 @@
-name: Test Only
+name: Lint
 
-# manual run, test only without linter
-# on: 
-#   workflow_dispatch:
-#     inputs:
-#       logLevel:
-#         description: 'Log level'     
-#         required: true
-#         default: 'warning'
-#       tags:
-#         description: 'Test scenario tags'
-# temporary use this, manual run button doesn't appear at actions tab
-on: [pull_request]
+on:
+  push:
+    branches: [master]
+  pull_request:
+  release:
+    types: [published, edited]
 
 jobs:
   tests:
```

Comment:
probably you could rename the job to `lint` for the correct description

Reply:
oh right thanks